apix-integrationtests/src/main/java/eu/xenit/apix/rest/v1/tests/DictionaryTest.java

```diff
@@ -147,4 +150,24 @@ public void missingDocumentType_status_404() throws IOException {
         assertEquals(404, httpResponse.getStatusLine().getStatusCode());
     }
 
+    public void assertMandatoryAspects(JSONObject jsonObject, String expectedMandatoryAspect) {try {
+        JSONArray jsonArray = jsonObject.getJSONArray("mandatoryAspects");
+        boolean aspectFound = false;
+        int jsonArrayLength = jsonArray.length();
+        int arrayIndex = 0;
+        while (!aspectFound & arrayIndex < jsonArrayLength) {
+            String element = (String) jsonArray.get(arrayIndex);
```

Comment:
Using https://stleary.github.io/JSON-java/org/json/JSONArray.html#getString-int- avoids the (String) cast.